satellite/console/consoleweb/server.go

```diff
@@ -190,68 +190,61 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 	var err error
 	defer mon.Task()(&ctx)(&err)
 
-	var projectID *uuid.UUID
-	var since, before time.Time
-
-	tokenCookie, err := r.Cookie("tokenKey")
+	host, _, err := net.SplitHostPort(r.Host)
 	if err != nil {
 		server.log.Error("bucket usage report error", zap.Error(err))
```

Comment:
Lower the level here to `Debug` since this is a caller error, not the server error.

Reply:
Well, if the parsed `request` has invalid `host`, but we reach this handler than I guess server might work incorrectly

Reply:
I'm not sure whether the http.Server verifies whether the `Host` field is a valid `HostPort` or not.

Reply:
And that is not `request.URL.Host`, which comes from the client

Reply:
```
// For server requests, Host specifies the host on which the URL
// is sought. Per RFC 7230, section 5.4, this is either the value
// of the "Host" header or the host name given in the URL itself.
// It may be of the form "host:port". 
```

Reply:
So my idea is that if `host` is incorrect(in come way), than `server` has some errors parsing the `request` so it's `InternalServerError`

If it is not valid `HostPort`, than we parse this field incorrectly, which is `InternalServerError` as well

Reply:
Ok, yeah, it looks like it ensures that it's a valid host:

```
	// Find the target host. Prefer the Host: header, but if that
	// is not given, use the host from the request URL.
	//
	// Clean the host, in case it arrives with unexpected stuff in it.
	host := cleanHost(r.Host)
	if host == "" {
		if r.URL == nil {
			return errMissingHost
		}
		host = cleanHost(r.URL.Host)
	}
```